docs/src/components/DocSearch.js

```diff
@@ -1,30 +1,18 @@
 // @flow strict
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
 import './DocSearch.css';
 
 export default function DocSearch() {
-  const [enabled, setEnabled] = useState(true);
-
   useEffect(() => {
-    // Disable search locally
```

Comment:
Also removed this - I didn't see this issue locally because of this conditional.